UI/src/app/components/dialog/dialog.component.html

```diff
@@ -1,45 +1,42 @@
 <mat-card class="dialog">
   <mat-list>
-    <ng-container *ngFor="let comment of comments">
-      <ng-template 
-      [ngTemplateOutlet]="loadTemplate(comment)" 
-      [ngTemplateOutletContext]="{ $implicit: comment}">
+    <ng-container *ngFor='let comment of comments'>
+      <ng-template [ngTemplateOutlet]='loadTemplate(comment)' [ngTemplateOutletContext]='{ $implicit: comment}'>
       </ng-template>
     </ng-container>
   </mat-list>
-
-  <input [(ngModel)]="value" [disabled]="isDisabled()" placeholder="Input comment" class="input input-add" (keyup.enter)="add()">
-
-  <button mat-button (click)="add()" [disabled]="isDisabled()" class="button-add" disableRipple="true">
+  <input [(ngModel)]='value' [disabled]='isDisabled()' placeholder='Input comment' class='input input-add'
+    (keyup.enter)='add()'>
+  <button mat-button (click)='add()' [disabled]='isDisabled()' disableRipple='true'>
     Add
   </button>
 </mat-card>
 
 <ng-template #readOnlyTemplate let-comment>
-  <mat-list-item class="list-item-read">
-    <div class="comment-read">
-      <div class="comment-read-head">
-        <span class="comment-read-date">
+  <mat-list-item>
+    <div class="read">
```

Comment:
now markup is cleaner, and we can use html5 semantic tags.
I suggest refactor this to 
```html
   <section class="read"> //section, header, main - semantic html5 tags
      <header>
         <span class="date">{{comment.date | date:'MMM d'}}
         <span  *ngIf="comment.hasBeenEdited">(edited)</span> //no need in additional ng-container for ng-if
      </header>
   </section>
```